### DISCUSSION TITLE
Prep for release. (Maintenance vs fork etc)

### DISCUSSION THREAD

**Discussion by andrevdm** [andrevdm/postgresql-migration#4] [General]
I'm planning to either take over maintenance of https://github.com/ameingast/postgresql-simple-migratio if @ameingast wants to hand it over, or otherwise keep this fork and release a new hackage package.

I use postgresql-simple-migration both for private and work projects and now that ameingast has archived the original project I think its worth having a fork to keep it alive.

What I have already done is

1. Merged my [PR:30](https://github.com/ameingast/postgresql-simple-migration/pull/30) to allow a custom schema_migrations table name
2. Merged [PR:36](https://github.com/ameingast/postgresql-simple-migration/pull/36) implementing a log redirection. From @unclechu
3. Implemented [Issue:40](https://github.com/ameingast/postgresql-simple-migration/issues/40) as requested by @clementd-fretlink
4. Relaxed bounds where appropriate
5. Moved to github actions rather than travis 

To do all of this there are unfortunately some breaking API changes. Most notably the change from **MigrationContext** to **MigrationOptions**. While I would have liked to have kept the API the same I believe that implementing PRs 30 and 36 required an API change to keep things sane. 

Given that the breaking change I also decided to remove support for older GHC versions, which cleans up the code a bit. 

[Milestone 1](https://github.com/andrevdm/postgresql-simple-migration/milestone/1) contains the issues I think need to be resolved before I can release the first version of the "fork".

_(My apologies for the @ if you are not interested in the project/fork. I wont @ again unless you explicitly participate from now on)_

Feel free to discuss any concerns or anything you'd like to see here.

**Comment by andrevdm** [comment]
Initial changes are on branch [feature/v2](https://github.com/andrevdm/postgresql-simple-migration/tree/feature/v2)

**Reply by andrevdm** [reply]
merged to master

**Comment by andrevdm** [comment]
I've added `Database.PostgreSQL.Simple.Migration.V1Compat` so that existing users can easily use their existing code with no changes apart from the imported module name. 

I've also added the exact v1 tests and they are all passing

**Comment by unclechu** [comment]
That’s nice to hear! Do you have any plans for a release of a new version?

**Reply by andrevdm** [reply]
I just need to create a few unit tests for the new transaction control feature. I ran out of time over the weekend, but this should be pretty quick. I'd like to have a release out before the end of this week if possible.

**Reply by andrevdm** [reply]
Oh and also a new name that is better than postgresql-simple-migration2. That is the real work now :grimacing:

**Reply by andrevdm** [reply]
Published to hackage: https://hackage.haskell.org/package/postgresql-migration-0.2.1.2

**Comment by andrevdm** [comment]
Ready for submission to [hackage](https://github.com/andrevdm/postgresql-migration/issues/7) and [stackage](https://github.com/andrevdm/postgresql-migration/issues/8) now. I've settled on the name `postgresql-migration` not terribly creative but searchable at least. 

All other [milestone 1](https://github.com/andrevdm/postgresql-migration/milestone/1) issues are done

**Comment by andrevdm** [comment]
https://hackage.haskell.org/package/postgresql-migration-0.2.1.2
